recipe/meta.yaml

```
@@ -76,6 +76,9 @@ requirements:
   run_constrained:
     - khronos-opencl-icd-loader  ==9999999999  # [osx and opencl_impl == 'apple']
     - ocl_icd_wrapper_apple      ==9999999999  # [osx and opencl_impl == 'apple']
+{% if py == 39 %}
+    - python <3.9.9  # [win]
+{% endif %}
```

This should be a run requirement instead of `run_constrained`.

```suggestion
    - python <3.9.9  # [py == 39]
```

---

Wouldn't that affect all platforms rather than just Windows?

---

Ah yes. `python <3.9.9  # [py == 39 and win]`